canonical-web-and-design/microk8s.io
# Rebuild microk8s home page

## Done

- Rebuild microk8s.io based on copy doc 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page matches [copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#)
- Check design 

## Issue / Card

Fixes [#3259](https://github.com/canonical-web-and-design/web-squad/issues/3259)
